components/_share.scss

```diff
@@ -0,0 +1,170 @@
+.sharing-info {
+  clear: both;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets